test/style_test.css

```diff
@@ -1,4 +1,4 @@
 GtkLabel {
   color: blue;
-  font: 16px;
+  font-size: 16px;
```

Comment:
Travis' gtk can't parse `font-size` for some reason, which is why I used just `font`. (https://github.com/JuliaGraphics/Gtk.jl/pull/204#issuecomment-160483195) I did forget to add the font-family value, sorry :P